cypress/integration/Header_spec.js

```diff
@@ -0,0 +1,8 @@
+describe('Header shows at the top of the home page', () => {
```

Comment:
This is a nice baseline. We can flesh out this test after we add the routing functionality/create the other components/views.